rosdep/python.yaml

```diff
@@ -6382,6 +6382,15 @@ python3-filterpy-pip:
   ubuntu:
     pip:
       packages: [filterpy]
+python3-fiona:
+  arch: [python-fiona]
+  debian: [python3-fiona]
+  fedora: [python3-fiona]
+  nixos: [python3Packages.fiona]
```

Comment:
How do `nixos` rules work? I see `python39Packages.fiona` and `python38Packages.fiona` but no `python3Packages.fiona`.

https://search.nixos.org/packages?from=0&size=50&type=packages&query=fiona

I wasn't able to figure it out from the rosdep code for nix either: https://github.com/ros-infrastructure/rosdep/blob/4b6a4f096513fa5356a864539c280a95249713ec/src/rosdep2/platforms/nix.py

Reply:
I asked the same thing a while ago, I cannot find the comment on github but those are aliases.

From https://github.com/NixOS/nixpkgs/blob/master/doc/languages-frameworks/python.section.md#building-packages-and-applications-building-packages-and-applications:

> pkgs.python2Packages pointing to pkgs.python27Packages
  pkgs.python3Packages pointing to pkgs.python39Packages
  pkgs.pythonPackages pointing to pkgs.python2Packages